deploy.Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+pipeline {
+    agent {label 'docker'}
+
+    stages {
+        stage('Pre Build ~ Clean Up Builder') {
+            steps {
+                sh 'docker system prune -f'
```

Comment:
are these prunes to help with space? since it seems that the tests are passing now?

Reply:
yes